protocol_details/asymmetric_mp.rst

```diff
@@ -25,4 +25,4 @@ Other reasons to run heterogeneous software environments (e.g. multi-OS) include
 
 In AMP systems, it is typical for software running on a master to bring up software/firmware contexts on a remote on a demand-driven basis and communicate with them using IPC mechanisms to offload work during run time. The participating master and remote processors may be homogeneous or heterogeneous in nature.
 
-A master is defined as the CPU/software that is booted first and is responsible for managing other CPUs and their software contexts present in an AMP system. A remote is defined as the CPU/software context managed by the master software context present.
\ No newline at end of file
+A master is defined as the CPU/software that is booted first and is responsible for managing other CPUs and their software contexts present in an AMP system. A remote is defined as the CPU/software context managed by the master software context present.
```

Comment:
Is there a white space difference here? I don't see any difference in the wording.